addons/html_editor/static/src/others/embedded_components/plugins/video_plugin/video_plugin.js

```diff
@@ -44,13 +45,32 @@ export class VideoPlugin extends Plugin {
         });
         this.dependencies.dom.insert(videoBlock);
         this.dependencies.history.addStep();
+        this.dependencies.selection.focusEditable();
+    }
+
+    /**
+     * @param {Object} props
+     */
+    extendEmbeddedVideoProps({ props }) {
+        if (props.host?.dataset.embedded === "video") {
```

Comment:
you receive `name` as an argument, which is guaranteed to be provided, so you should do this instead:
```suggestion
    extendEmbeddedVideoProps({ name, props }) {
        if (name === "video") {
```